api/serializers/color_serializer.py

```diff
@@ -0,0 +1,10 @@
+from retail_app.models import ProductColor
+import json
+
+
+class ColorSerializer:
+    def __init__(self, color: ProductColor):
+        self.color = color
+
+    def for_json(self):
+        return self.color.color
```

Comment:
Using the `to_json` method name is more conventional, like `toString` when doing conversions. If you keep with the `for` naming, I would make it `json_for`